docs/features/theme.stories.mdx

```diff
@@ -178,36 +178,31 @@ theme.borderWidth.[kilo|mega]
 ## Typography
 
 ```
-theme.typography.[headings|subHeadings|text].<SIZE_NAME>.[fontSize|lineHeight]
+theme.typography.[headlines|subHeadlines|text].<SIZE_NAME>.[fontSize|lineHeight]
 ```
 
-Avoid using `theme.typography` directly in your styles, rather use the specialized components [`Heading`](Typography/Heading/Base), [`SubHeading`](Typography/SubHeading/Base), and [`Text`](Typography/Text/Base).
+Avoid using `theme.typography` directly in your styles, rather use the specialized components [`Headline`](Typography/Headline/Base), [`SubHeadline`](Typography/SubHeadline/Base), and [`Body`](Typography/Body/Base).
 
-### Headings
+### Headlines
 
 <Preview>
-  <Type size="zetta" component={Heading} name="headings" />
-  <Type size="exa" component={Heading} name="headings" />
-  <Type size="peta" component={Heading} name="headings" />
-  <Type size="tera" component={Heading} name="headings" />
-  <Type size="giga" component={Heading} name="headings" />
-  <Type size="mega" component={Heading} name="headings" />
-  <Type size="kilo" component={Heading} name="headings" />
+  <Type size="one" component={Headline} name="headings" />
+  <Type size="two" component={Headline} name="headings" />
+  <Type size="three" component={Headline} name="headings" />
+  <Type size="four" component={Headline} name="headings" />
```

Comment:
The `name` prop refers to the theme property, e.g. `headline` in `theme.typography.headline.one.fontSize`. Thus, the value should be `headline` here:
```suggestion
  <Type size="one" component={Headline} name="headline" />
  <Type size="two" component={Headline} name="headline" />
  <Type size="three" component={Headline} name="headline" />
  <Type size="four" component={Headline} name="headline" />
```